Core/Graphics/Include/Shared/Babylon/Graphics/Device.h

```diff
@@ -9,6 +9,11 @@
 
 namespace Babylon::Graphics
 {
+    struct PlatformInfo
+    {
+        DeviceType Device;
```

Comment:
nit: This might be just me, but I don't really like `DeviceType` as a type alias because it sounds like an enum, such that I might expect to see something like
```
DeviceType Device{ DeviceType::D3D12 };
```
The convention I prefer is `DeviceT` for type aliases as that invokes template conventions, which I feel is more similar to what we're trying to express here than an enum is. Not a huge deal, but it confuses me ever time I encounter something like this.

Reply:
I'm okay with that, but let's save it for a different PR.